### DISCUSSION TITLE
Where is the changelog?

### DISCUSSION THREAD

**Discussion by megatux** [paperwm/PaperWM#915] [Q&A] [answered]
Hi, this extension is awesome, probably the only reason I still use Gnome.
I see the notification for new version every couple of days, great! but where is the list of changes? I do not see a GHANGELOG file in the repo or similar. Is there a place where I can see the list of bug-fixes, new features, changes, etc added on every new release?  Thanks!

**Comment by Lythenas** [comment, marked as answer]
The github releases at least show the list of all PRs that were merged for that release: https://github.com/paperwm/PaperWM/releases

We don't write any more detailed changelog.
